docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Platform
+sidebar_label: Platform
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+Every robot platform has unique structures such as versatile sensor mounting solutions, wireless charging receivers, and waterproofing enclosures; we refer to these as attachments.
+
+## Joystick Controller
+We support two types of controllers:
+  - **ps4**: standard Playstation4 controller.
+  - **logitech**: Logitech F710
```

Comment:
Do we still support Logitech? I thought these were deemed problematic.

Reply:
Yes, we support it in the software. I'm not sure if we decided to not go with these controllers.

Reply:
Okay, thanks for the info. We should keep it in the documentation website for now.